stl/inc/algorithm

```diff
@@ -2579,14 +2579,14 @@ void _Push_heap_by_index(_RanIt _First, _Iter_diff_t<_RanIt> _Hole, _Iter_diff_t
     // percolate _Hole to _Top or where _Val belongs, using _Pred
     using _Diff = _Iter_diff_t<_RanIt>;
     for (_Diff _Idx = (_Hole - 1) >> 1; // shift for codegen
-         _Top < _Hole && _DEBUG_LT_PRED(_Pred, *(_First + _Idx), _Val); //
+         _Top < _Hole && _DEBUG_LT_PRED(_Pred, _First[_Idx], _Val); //
          _Idx = (_Hole - 1) >> 1) { // shift for codegen
         // move _Hole up to parent
-        *(_First + _Hole) = _STD move(*(_First + _Idx));
+       _First[_Hole] = _STD move(_First[_Idx]);
         _Hole             = _Idx;

```

Comment:
To pass clang-format 9.0.0 validation, the `_Hole             = _Idx;` assignment needs to have its whitespace adjusted, so that the assignment is aligned with the line that you edited. (If you look at the Azure Pipelines failure, it runs clang-format followed by `git diff` so you can see the edits that it's "requesting". Click on the "Details" for "microsoft.STL (x64)".)

We recommend downloading and running clang-format to make these changes automatically. Your editor may be capable of running clang-format on save. (We've updated the repo so that if you open it in VSCode, and have the VSCode C/C++ Extension installed and configured with clang-format 9.0.0, it will Format On Save even if you don't normally have that option enabled.) However, if you want to make such edits manually until the validation passes, that is workable.

Reply:
Thank you for your reply, I'll fix this today